eng/ignore-links.txt

```diff
@@ -5,3 +5,5 @@ https://github.com/Azure/azure-sdk-for-net-pr/tree/feature/IoT-ADT/sdk/digitaltw
 https://github.com/Azure/azure-sdk-for-net-pr/blob/feature/IoT-ADT/sdk/iot/Azure.Iot.DigitalTwins/samples/DigitalTwinServiceClientSample/DTDL/Relationships/HospitalRelationships.json
 https://www.nuget.org/packages/Azure.ResourceManager.DigitalTwins
 https://contoso.azureedge.net/urlsigning/test?expires=2145916800&amp;keyid:key1&amp;signature=iTsrLX9rVAIJkSahBA_j5o9Azf5-j331ohxDR1Gx2js=
+https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal
```

Comment:
Why are we adding these links to the ignore list? We should not have broken links in our readmes. See http://aka.ms/azsdk/guideline/links.

Reply:
`https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal` is not a broken link but the pipeline was failing with broken links error(check [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=855903&view=logs&j=36df4387-3551-57fa-098c-b43a23930261&t=3851d55a-4253-57f5-016e-927e8de9d21b&l=19)). I contacted @scbedd, he said 
>I'm going to need to follow up with the script writers. I've been looking at this backwards and forwards and I still don't know why it's failing the check.

To unblock this PR he suggested to add these links in `eng/ignore-links.txt` file while he is looking into the issue.

Reply:
```
##[warning]DO NOT include locale /en-us/ information in links: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal. Check here for more information: https://aka.ms/azsdk/guideline/links
##[warning]DO NOT include locale /en-us/ information in links: https://docs.microsoft.com/en-us/azure/cognitive-services/translator/document-translation/create-sas-tokens?tabs=Containers. Check here for more information: https://aka.ms/azsdk/guideline/links
```

The warning tells you what to do and provide a link for more information.

Reply:
Thanks for the advice. The /en-us/ is removed from the links.